test/WorkflowCore.Testing/JsonWorkflowTest.cs

```diff
@@ -39,6 +39,25 @@ protected virtual void Setup()
             Host.Start();
         }
 
+        protected virtual void Setup(IServiceCollection services)
```

Comment:
Could you not just override the `ConfigureServices` method in your tests?

Reply:
Yes, I see that I can override ConfigureServices, but the services variable declared in Setup() is a local variable. The Host is started using the local variable. I don't see a way by which I could pass my IServiceCollection, before the Host is started.

Reply:
You don't need to pass your own instance of IServiceCollection... you can interact with it in the `ConfigureServices` method